taxonomy-manager-rest-server/src/main/java/com/digirati/taxman/rest/server/infrastructure/event/ConceptEventListener.java

```diff
@@ -1,18 +1,52 @@
 package com.digirati.taxman.rest.server.infrastructure.event;
 
 import com.digirati.taxman.analysis.index.TermIndex;
+import com.digirati.taxman.common.taxonomy.ConceptLabelExtractor;
+import com.digirati.taxman.common.taxonomy.ConceptModel;
+import com.google.common.eventbus.Subscribe;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
+import java.util.Collection;
 import java.util.UUID;
+import java.util.function.BiConsumer;
 
 /**
  * Listener to respond to changes in the state of a concept.
  */
 @ApplicationScoped
 public class ConceptEventListener {
 
+    @ConfigProperty(name = "taxman.analysis.default-lang.key", defaultValue = "en")
+    String defaultLanguageKey;
+
     @Inject
     TermIndex<UUID> index;
 
+    @Subscribe
+    public void ConceptChangeEvent(ConceptEvent conceptEvent) {
```

Comment:
Style  should be consistent with the rest of the project (braces on even 1-statement branches, camelCase members, PascalCase nominals).

Reply:
.NET has leaked in ;)